src/main/java/com/sumologic/client/collectors/model/StreamingMetricsSource.java

```diff
@@ -0,0 +1,58 @@
+package com.sumologic.client.collectors.model;
+
+public class StreamingMetricsSource extends Source {
+
+    private static String PROTOCOL = "protocol";
+    private static String PORT = "port";
+    private static String CONTENT_TYPE = "content_type";
```

Comment:
content type should be a common field for all sources (see `Sources.scala` in collector-interchange), so I think we should move it to the base class `Source` in the java client as well https://github.com/SumoLogic/sumo-java-client/blob/master/src/main/java/com/sumologic/client/collectors/model/Source.java

the reason it was not in there already is probably because we don't need this field for other sources in the java client (it's mostly used by hosted sources, like AWS Cloudwatch and http metrics etc), and streaming metrics source is the first one in the java client that actually uses it

Reply:
Agreed. I was thinking about that initially but was hesitant to change the base class. Done now.